authors/apps/articles/test/test_create_article.py

```diff
@@ -0,0 +1,130 @@
+import json
+from django.urls import reverse
+from rest_framework.views import status
+from rest_framework.test import APITestCase, APIClient
+# from ..models import User
+# from ..models import Articles
+from authors.apps.articles.models import Article
+# from authors.apps.authentication.views import LoginAPIView, RegistrationAPIView
+
+class CreateArticles(APITestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.login_url = reverse('authentication:auth-login')
+        self.signup_url = reverse('authentication:auth-register')
+        self.listcreate = reverse('articles:articles-listcreate')
+        self.signup_data = {
+            "user": {
+                "username": "kennyg",
+                "email": "ken@me.com",
+                "password": "Kennyisme1!"
+                }}
+        self.login_data = {
+            "user": {
+                "email": "ken@me.com",
+                "password": "Kennyisme1!"
+                }}
+        self.signup_data2 = {
+            "user": {
+                "username": "kwammea",
+                "email": "kwame@me.com",
+                "password": "Kwameisme1!"
+                }}
+        self.create_article_data = {
+            "title": "A new story",
+            "body": "This is my story",
+            "description": "Here is my story",
+            "images": None
+        }
+        self.create_article_data2 = {
+            "title": "The beginning",
+            "body": "This begins nowy",
+            "description": "Here is my story",
+            "images": None
+        }
+        self.delete_article = {
+            "is_displayed": False
+        }
+
+    def test_fetch_all_articles(self):
+        """Tests to fetch all articles"""
+        register = self.client.post(self.signup_url,
```

Comment:
You can create a method that returns a token then call it to avoid repetition

Reply:
Noted